src/lib/config-utils.ts

```diff
@@ -187,6 +310,42 @@ function setRpiBootConfig(target: ConfigOptions): Promise<void> {
 	return remountAndWriteAtomic(getRpiBootConfig(), confStatements.join('\n') + '\n');
 }
 
+function setExtlinuxBootConfig(target: ConfigOptions): Promise<void> {
+	// First get a representation of the configuration file, with all resin-supported configuration removed
+	return Promise.resolve(fs.readFile(getExtlinuxBootConfig()))
```

Comment:
I don't see the benefit of using `mz.fs` here.
You can use `Bluebird.promisify` for `Bluebird` promises or `util.promisify` for regular promises.
Here you use `mz.fs` methods which return regular promises that you turn into `Bluebird` promises with `Bluebird.resolve`.

Reply:
There's two reasons I used `mz`, one is that I was getting several issues with types when using `promisify`, where the typescript compiler was expecting different parameter counts etc, but more importantly it makes the `stub`bing of these functions in the test suite much easier - that's not to say it can't be done using promisification, I'd prefer this to be hanlded elsewhere, as it's outside of the scope of the supervisor.